fmriprep/workflows/bold/resampling.py

```diff
@@ -280,13 +282,10 @@ def init_bold_std_trans_wf(
         Size of BOLD file in GB
     omp_nthreads : int
         Maximum number of threads an individual process may use
-    standard_spaces : OrderedDict
-        Ordered dictionary where keys are TemplateFlow ID strings (e.g.,
-        ``MNI152Lin``, ``MNI152NLin6Asym``, ``MNI152NLin2009cAsym``, or ``fsLR``),
-        or paths pointing to custom templates organized in a TemplateFlow-like structure.
-        Values of the dictionary aggregate modifiers (e.g., the value for the key ``MNI152Lin``
-        could be ``{'resolution': 2}`` if one wants the resampling to be done on the 2mm
-        resolution version of the selected template).
+    std_references : :obj:`list` of :obj:`tuple`
```

Comment:
isn't this a `SpatialReference` object? its `get_template()` method is being used below.